src/components/spinner/spinner.css

```diff
@@ -1,3 +1,11 @@
+.loader.loader--full-screen {
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+}
```

Comment:
jeg tænker der er brug for et z-index her. for ellers vil mange ting gå henover?

Reply:
Det er korrekt. Har i denne omgang bare flyttet eksisterende kode. Kan godt tilføje z-index, skulle der evt. også noget background: rgba(0,0,0,0.5) på, så man kan fornemme loaderen?